src/main/java/pp/javastart/budget/BudgetUpdate.java

```diff
@@ -0,0 +1,49 @@
+package pp.javastart.budget;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+public class BudgetUpdate {
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Podaj id transakcji, którą chcesz zaktualizować");
+        long id = scanner.nextLong();
+        scanner.nextLine();
+
+        System.out.println("Podaj typ: wydatek/przychód");
+        TransactionType type;
+        type = TransactionType.getByName(scanner.nextLine());
+        while (!TransactionType.SPEND.equals(type) && !TransactionType.INCOME.equals(type)) {
+            System.out.println("Zła nazwa wydatku: poprawne wartości to 'wydatek' lub 'przychód'");
+            type = TransactionType.getByName(scanner.nextLine());
+        }
+        System.out.println("Podaj opis");
+        String description = scanner.nextLine();
+
+        System.out.println("Podaj kwote transakcji");
+        BigDecimal amount = scanner.nextBigDecimal();
+        scanner.nextLine();
+
+        DateTimeFormatter dateFormater = DateTimeFormatter.ofPattern("yyyy-MM-dd");
```

Comment:
Tutaj to samo co wcześniej - może to być stała

Reply:
done